_posts/2022-11-07-what-makes-a-python-package-healthy.md

```diff
@@ -0,0 +1,559 @@
+
+**My goal:** to see what the community thought about "*what constitutes package health*". 
+
+<blockquote class="twitter-tweet" data-partner="tweetdeck"><p lang="en" dir="ltr">controversial topic: How do we measure the &quot;health&quot; of a <a href="https://twitter.com/hashtag/science?src=hash&amp;ref_src=twsrc%5Etfw">#science</a> <a href="https://twitter.com/hashtag/python?src=hash&amp;ref_src=twsrc%5Etfw">#python</a> package? GitHub stars? downloads, date of latest commit? # of commits a month / quarter? Spread of commits? Thoughts? <a href="https://twitter.com/hashtag/opensource?src=hash&amp;ref_src=twsrc%5Etfw">#opensource</a> <a href="https://twitter.com/hashtag/OpenScience?src=hash&amp;ref_src=twsrc%5Etfw">#OpenScience</a> <a href="https://twitter.com/pyOpenSci?ref_src=twsrc%5Etfw">@pyOpenSci</a></p>&mdash; Leah Wasser 🦉 (@LeahAWasser) <a href="https://twitter.com/LeahAWasser/status/1577730887818498049?ref_src=twsrc%5Etfw">October 5, 2022</a></blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The twitter convo made me realize that there are
+many different perspectives that we can consider when addressing this question. 
+
+More specifically, pyOpenSci is interested in the health of packages that
+support science. So we may need to build upon existing 
+efforts that have determined what metrics to use to quantify package health and customize them to our needs.
+
+<div class="notice" markdown="1">
+### A note about our pyOpenSci packages 
+
+pyOpenSci does not focus on foundational scientific Python packages like Xarray 
+Dask, or Pandas. Those packages are stable and already have a large user base 
+and maintenance team. Rather we focus on packages that are higher up in the 
+ecosystem. These packages tend to have smaller user bases, and smaller maintainer 
+teams (or often are maintained by one volunteer person).
+
+<figure>
+    <a href="/images/python-packages/python-ecosystem-jake-vanderploss-pycon-2017.png"><img src="/images/python-packages/python-ecosystem-jake-vanderploss-pycon-2017.png" alt="Image used by Jake Vanderploss in the 2017 pycon conference that shows 
+    the ecosystem of scientific python packages starting with foundational packages 
+    and moving out to the wealth of smaller, domain specific packages."></a>
+    <figcaption>Image used by Jake Vanderploss in the 2017 pycon conference that shows 
+    the ecosystem of scientific python packages starting with foundational packages 
+    and moving out to the wealth of smaller, domain specific packages. pyOpenSci
+    focuses on the packages that are higher up in the ecosystem that often could 
+    benefit from more support. 
+    </figcaption>
+</figure>
+
+Our package maintainers:
+
+* Often don't have the resources to build community
+* Often are keen to build their user base and to contribute
+to the broader scientific python ecosystem.   
+</div>
+
+
+### Three open source software healthy metric "buckets" 
```

Comment:
the flow here loses me a little -- I think maybe you meant to remove this? since you repeat it below  

If this is meant to be a callback to previous posts, I would make that explicit.  

And/or insert a transition sentence to signal to your reader that you're not going to talk about these things yet: "Of course, many people have thought about metrics for package health, and this previous work came up in the conversation on twitter. Let me share those and acknowledge that work before I say more about this..."